csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt

```diff
@@ -14,19 +14,19 @@ id:ID(TagClass)|name:STRING|url:STRING
 sf-003/parquet/31/part.0.parquet
 sf-003/parquet/31/part.1.parquet
 
-creationDate:IGNORE|id:ID(Comment)|locationIP:STRING|browserUsed:STRING|content:STRING|length:STRING
+creationDate:DATETIME|id:ID(Comment)|locationIP:STRING|browserUsed:STRING|content:STRING|length:STRING
 sf-003/parquet/32/part.0.parquet
 sf-003/parquet/32/part.1.parquet
 
-creationDate:IGNORE|id:ID(Forum)|title:STRING
+creationDate:DATETIME|id:ID(Forum)|title:STRING
 sf-003/parquet/33/part.0.parquet
 sf-003/parquet/33/part.1.parquet
 
-creationDate:STRING|id:ID(Person)|firstName:STRING|lastName:STRING|gender:STRING|birthday:STRING|locationIP:STRING|browserUsed:STRING
+creationDate:DATETIME|id:ID(Person)|firstName:STRING|lastName:STRING|gender:STRING|birthday:STRING|locationIP:STRING|browserUsed:STRING
 sf-003/parquet/34/part.0.parquet
 sf-003/parquet/34/part.1.parquet
 
-creationDate:IGNORE|id:ID(Post)|imageFile:STRING|locationIP:STRING|browserUsed:STRING|language:STRING|content:STRING|length:STRING|forumID:STRING
+creationDate:DATETIME|id:ID(Post)|imageFile:STRING|locationIP:STRING|browserUsed:STRING|language:STRING|content:STRING|length:STRING|forumID:IGNORE
```

Comment:
based on the parts it is hard to know what is in the files; we should have a comments or labels what nodes are being here described.

Reply:
@danielmawhirter any idea how we could map each of these parquet files to the corresponding node/ edge it belongs to?